src/libs/Url.ts

```diff
@@ -1,25 +1,16 @@
 import 'react-native-url-polyfill/auto';
-/**
- * Add / to the end of any URL if not present
- * @param {String} url
- * @returns {String}
- */
-function addTrailingForwardSlash(url) {
+
+function addTrailingForwardSlash(url:string):string {
```

Comment:
```suggestion
function addTrailingForwardSlash(url:string): string {
```

Can you run `npm run prettier`? I'm not sure if it supports such spacing but a space look better here

Reply:
Yeah you aer right